openedx/features/student_certificates/tasks.py

```diff
@@ -0,0 +1,40 @@
+# import imgkit
+from os import remove
+from logging import getLogger
+
+from django.conf import settings
+from celery.task import task
+
+
+from helpers import upload_to_s3, get_certificate_url, get_certificate_image_name, \
+    get_certificate_img_key
+
+log = getLogger(__name__)
+
+
+@task(routing_key=settings.HIGH_MEM_QUEUE)
+def task_create_certificate_img_and_upload_to_s3(certificate):
+    """
+    :param certificate:
+    :return:
+    """
+    try:
+        url = get_certificate_url(certificate)
+        img_name = get_certificate_image_name(certificate)
+        uuid = certificate.verify_uuid
```

Comment:
let's move this above and pass this id in helper methods and modify them as well.